openlibrary/plugins/upstream/utils.py

```diff
@@ -650,7 +650,10 @@ def get_donation_include(type):
         dd = input['don']
```

Comment:
Not apart of your this review, but (a) we seem to be overriding another builtin function `input` (i.e. https://docs.python.org/2/library/functions.html#input) and it's unclear what happens when 'don' is an undefined key. Perhaps it should default to '', i.e. dd = inputs.get('don', '')